frontend/components/SocialMediaShareIcon.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="cursor-pointer"
+    style="width: 100%; height: 100%; justify-content: flex-start; align-items: center; gap: 12px; display: inline-flex">
+    <div style="iconStyle">
+      <svg width="32" height="32" viewBox="0 0 16 16" :fill="fillColor" xmlns="http://www.w3.org/2000/svg">
+        <path :d="iconPath" />
+      </svg>
+    </div>
+    <div>{{ text }}</div>
+  </div>
+</template>
+  
+<script>
```

Comment:
Can we get this to be `scribe setup lang="ts"` so that we're using the composition API and TypeScript?

Reply:
Updated, just gotta fix up a couple of errors that popped up. I looked at the other comments and I believed I fixed those

Reply:
Kay I have everything updated and working. I'm going to wait until the dev sync tomorrow to push things up b/c I now see this when I attempt to push to my branch.  It's also saying I have 124 changes to sync. So I'll hold off for now.

```
miss_tessa@miss_tessa-wimg-remote-mac activist-TA % git push
To github.com:GrandSuccess87/activist-TA.git
 ! [rejected]        #681/feature-Create-Modal-Share-Page-Component -> #681/feature-Create-Modal-Share-Page-Component (non-fast-forward)
error: failed to push some refs to 'github.com:GrandSuccess87/activist-TA.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
miss_tessa@miss_tessa-wimg-remote-mac activist-TA % git pull
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
miss_tessa@miss_tessa-wimg-remote-mac activist-TA % git merge main
Already up to date.
miss_tessa@miss_tessa-wimg-remote-mac activist-TA % 
```